core/controllers/dashboard_test.py

```diff
@@ -16,13 +16,22 @@
 
 from core.controllers import dashboard
 from core.domain import config_services
+from core.domain import event_services
 from core.domain import feedback_domain
 from core.domain import feedback_services
+from core.domain import rating_services
 from core.domain import rights_manager
+from core.domain import stats_jobs_continuous_test
 from core.domain import user_jobs_continuous
+from core.domain import user_jobs_continuous_test
 from core.tests import test_utils
```

Comment:
Ok, will fix. 

Just did a quick google search and found that the latest version of pylint now detects that the imports are in alphabetic ordering ->  [SO Answer](http://stackoverflow.com/a/35213338/4135178)

Should we consider updating our pylint to the latest version ?